GoogleChromeLabs/ps-analysis-tool
# Enhancement: CLI  output path relative to "out" directory.

## Description

This PR updates the behavior of creating directories for analysis output. All relative paths will be relative to "out" directory.


## Testing Instructions

Commands like `npm run cli -- -u https://www.aljazeera.com/ -d foo` or `npm run cli -- -u https://www.aljazeera.com/ -d ./foo` should make a directory relative to "out" directory, **not** the project root


## Checklist






Fixes #626
